modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -332,7 +333,8 @@ public class IgniteImpl implements Ignite {
                 clusterSvc,
                 raftMgr,
                 clusterStateStorage,
-                logicalTopology
+                logicalTopology,
+                nodeCfgMgr.configurationRegistry().getConfiguration(ClusterManagementConfiguration.KEY)
```

Comment:
Don't we have a variable that's equal to `nodeCfgMgr.configurationRegistry()`? Please check

Reply:
We don't, but we have

```
public ConfigurationRegistry nodeConfiguration() {
    return nodeCfgMgr.configurationRegistry();
}
```

in this class. At the same time, it seems awkward to use a public method in our own constructor, this seems a bit fragile.

So I just extracted 2 local variables in the constructor (one for node config registry, another for cluster config registry).

Reply:
Ok, thank you! I really thought that we should have a var like this